src/assets/scripts/client/game/GameController.js

```diff
@@ -421,6 +428,7 @@ class GameController {
         if (this.game.score === this.game.last_score) {
             return;
         }
+
         const $scoreElement = $(SELECTORS.DOM_SELECTORS.SCORE);
         $scoreElement.text(round(this.game.score));
```

Comment:
There should be a blank line...I think 😉

Reply:
correct again!